toolkit/tools/imagecustomizerschemacli/main.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/invopop/jsonschema"
+	"github.com/microsoft/azurelinux/toolkit/tools/imagecustomizerapi"
+)
+
+func main() {
+	if len(os.Args) < 1 {
```

Comment:
Is the `os.Args[0]` the binary name itself? Should here be:
```
if len(os.Args) < 2 {
...
}
```

Reply:
binary name doesn't count